picard/metadata.py

```diff
@@ -290,49 +320,62 @@ def clear_deleted(self):
         self.deleted_tags = set()
 
     def getall(self, name):
-        return super().get(name, [])
+        return self._store.get(name, [])
+
+    def getraw(self, name):
+        return self._store[name]
 
     def get(self, name, default=None):
-        values = super().get(name, None)
+        values = self._store.get(name, None)
         if values:
             return self.multi_valued_joiner.join(values)
         else:
             return default
 
+    def __contains__(self, name):
+        return self._store.__contains__(name)
+
     def __getitem__(self, name):
         return self.get(name, '')
 
     def set(self, name, values):
-        super().__setitem__(name, values)
-        if name in self.deleted_tags:
-            self.deleted_tags.remove(name)
+        self._store[name] = values
+        self.deleted_tags.discard(name)
 
     def __setitem__(self, name, values):
         if not isinstance(values, list):
             values = [values]
```

Comment:
Maybe we should accept any iterable here (so it can be converted to a list with `list(values)`, I recently had a case where it would have been convenient to pass a set.

Reply:
Yes, i thought about it, but decided to not modify this yet. I will do once we merged this.
Accepting any iterable would be much more logical (and convenient).